rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -73,13 +71,13 @@ void Player::wait_for_filled_queue(const Rosbag2PlayOptions & options) const
   }
```

Comment:
it's not in the diff, so I comment on the line below.
Didn't we say we want to put the 100 milliseconds delay as part of the `PlayOptions`? I would like to see this option set in the calling function and not in the implementation of it. I think it more transparent to expose them outside of the function.

Reply:
I am not sure that this constant is interesting as an cli option. I suggest we discuss thes also on #34.